bk-api/bk_api.py

```diff
@@ -903,7 +911,39 @@ def deploy_wes(node_id, this_debug, force=False):
     return {"success":True}
 
 
+def register_add_vsn_event(node_id,field_value, lock_tables=True, lock_requested_by=""):
+
+    payload = {"node_id": node_id, "source": "beekeeper-add-vsn", "operation":"insert", 
+                "field_name": "vsn", "field_value": field_value}
+
+    #url = f'{BEEKEEPER_DB_API}/log'
```

Comment:
remove

Reply:
Why?

Reply:
you need the commented out line of code?

```
#url = f'{BEEKEEPER_DB_API}/log'
```